docs/source/supported/anvil.rst

```diff
@@ -0,0 +1,117 @@
+================
+Anvil (Purdue)
+================
+
+Platform user guide
+===================
+
+https://www.rcac.purdue.edu/knowledge/anvil
+
+General description
+===================
+
+* Resource manager - ``SLURM``
+* Launch methods (per platform ID)
+
+  * ``purdue.anvil`` - ``SRUN``
+
+* Configuration per node
+
+  * ``debug`` queue (17 nodes)
+
+    * 256 CPU cores
+    * 257 GB of memory
+
+  * ``gpu-debug`` queue (16 nodes)
+
+    * 128 CPU cores
+    * 2 GPUs (per node)
+    * 515 GB of memory
+
+  * ``wholenode`` queue (750 nodes)
+
+    * 128 CPU cores
+    * 257 GB of memory
+
+  * ``wide`` queue (750 nodes)
+
+    * 128 CPU cores
+    * 257 GB of memory
+
+  * ``shared`` queue (250 nodes)
+
+    * 128 CPU cores
+    * 257 GB of memory
+
+  * ``highmem`` queue (32 nodes)
+
+    * 128 CPU cores
+    * 1031 GB of memory
+
+  * ``gpu`` queue (16 nides, max 12 GPUs per user)
+
+    * 128 CPU cores
+    * 4 GPUs (per node)
+    * 515 GB of memory
+
+
+
+Setup execution environment
+===========================
+
+Python virtual environment
+--------------------------
+
+Create a **virtual environment** with ``venv``:
```

Comment:
`venv` and `conda` as we have for Perlmutter https://github.com/radical-cybertools/radical.pilot/blob/devel/docs/source/supported/perlmutter.rst#setup-execution-environment